superset-frontend/src/SqlLab/components/AceEditorWrapper/useKeywords.ts

```diff
@@ -50,6 +50,11 @@ const EMPTY_LIST = [] as typeof sqlKeywords;
 const { useQueryState: useSchemasQueryState } = schemaEndpoints.schemas;
 const { useQueryState: useTablesQueryState } = tableEndpoints.tables;
 
+const getHelperText = (value: string) =>
+  value.length > 30 && {
```

Comment:
@michael-s-molina 
Unfortunately it's very complicated to make the tooltip dynamically. 
First, ace-editor doesn't provide the enough apis to handle the tooltip dynamically. (it's virtualized list so it should calculate the each item width dynamically as typed.)

To make the solution simple, it uses the static length since the tooltip width is fixed size (260px) and we can mostly guarantee the 30 characters are approaching the max. width(260px).

In my personal opinion, we can add `docText` for all cases to be consistence/simple for UX.